### DISCUSSION TITLE
Slowness of c7n log exporter tool on ECS batch

### DISCUSSION THREAD

**Discussion by sbanala12** [cloud-custodian/cloud-custodian#9239] [AWS]
We configured the c7n log exporter on ECS batch . When we execute the ECS batch , during execution observed notable slowness of transferring Cloudwatch log groups to S3 bucket. For 48 mb log group it took an long duration 4 hours to move the logs from cloudwatch to the s3 bucket.  Could assist with the cause for slowness.  I verified the ECS Batch Task CPU/memory usage is normal.

**Comment by kapilt** [comment]
we're just calling create_export_task on the aws api when manually syncing (also supports  a subscription output) re slowness is on the aws side in that context afaics.
